VideoEmbedFieldtype.php

```diff
@@ -97,7 +97,7 @@ public function process($data)
             if ($contents = curl_exec($request)) {
                 $contents = json_decode($contents, true);
                 $data['author_url'] = $contents['author_url'];
-                unset($data['duration']); // YouTube does not return duration so lets unset it
+                //unset($data['duration']); // YouTube does not return duration so lets unset it
```

Comment:
You are correct, this like removes the duration from YouTube videos as the endpoint I was using before did not return the duration. Thanks to your change it now does. So instead of commenting out this line it should be removed :)